TI-reports/2025/2025-Q1-Repos-WG.md

```diff
@@ -0,0 +1,52 @@
+# 2025 Q1 Securing Software Repositories Working Group
+
+## Overview
+
+**Mission**: Improve security of software repositories (npm, PyPI, RubyGems, ...) by providing a forum for discussion, a maturity model for security roadmaps, and guidance for individual security capabilities.
+
+**Links**:
+- [GitHub repository](https://github.com/ossf/wg-securing-software-repos)
+- [Slack channel](https://openssf.slack.com/archives/C034CBLMQ9G)
+- [WG meeting docs](https://docs.google.com/document/d/18Y8HxntL2RkcgqoFdhdLpj17e4MOSCdskP1IoDiuP1s/edit?usp=sharing)
+
+## Securing Software Repositories Working Group
+
+### Purpose
+
+Improve security of software repositories by providing a forum for discussion, a maturity model for security roadmaps, and guidance for individual security capabilities.
+
+### Current Status
+
+- [Central now performs Sigstore Signature Validation](https://central.sonatype.org/news/20250128_sigstore_signature_validation_via_portal/)
+- [Posting for technical writer](https://jobs.smartrecruiters.com/LinuxFoundation/744000038830864-openssf-securing-repositories-working-group-technical-writer) to write package yanking guidance is live
+- Letter of support to Python Software Foundation's grant request to US National Science Foundation on detecting, flagging, and quarantining malware
```

Comment:
This sentence doesn't actually say what the status is. Did the WG send that letter?

Reply:
Updated - we wrote and submitted a letter of support